<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HallOfFame</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>backgroundLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="heizhaoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>heizhaoLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="faguangLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>faguangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="guangxianLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>guangxianLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="xingxingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>xingxingLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="xunzhangLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>xunzhangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pianmingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>pianmingLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mingrentangLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mingrentangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jiLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>jiLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jinyuanLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>jinyuanLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="huizhanglishiLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>huizhanglishiLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="zhengzaijinxingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>zhengzaijinxingLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yueLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yueLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="zhouLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>zhouLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yuanguangTongLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yuanguangTongLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yuanguangJinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yuanguangJinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yinyuanLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yinyuanLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tongyuanLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>tongyuanLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jinkuangLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>jinkuangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yuanguangYinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yuanguangYinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tongkuangLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>tongkuangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="jinkuangxiaLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>jinkuangxiaLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qiehuantouxiangJinLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>qiehuantouxiangJinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yinkuangxiaLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yinkuangxiaLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yinkuangLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yinkuangLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tongkuangxiaLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>tongkuangxiaLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="touxiangYinLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>touxiangYinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin6Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin6Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="touxiangTongLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>touxiangTongLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qiehuantouxiangTongLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>qiehuantouxiangTongLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="touxiangJinLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>touxiangJinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qiehuantouxiangYinLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>qiehuantouxiangYinLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin4Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin4Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin3Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin3Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin2Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin2Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin1Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin1Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin5Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin5Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziJin7Label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziJin7Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin1Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin1Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin3Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin3Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin7Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin7Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin5Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin5Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin6Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin6Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin2Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin2Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziYin4Label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziYin4Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong1Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong1Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong3Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong3Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong7Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong7Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong5Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>350</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong5Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong6Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong6Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong2Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>290</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong2Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuziTong4Label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>shuziTong4Label</string>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>heizhaoLabel</zorder>
  <zorder>jinyuanLabel</zorder>
  <zorder>tongyuanLabel</zorder>
  <zorder>yinyuanLabel</zorder>
  <zorder>touxiangTongLabel</zorder>
  <zorder>touxiangYinLabel</zorder>
  <zorder>touxiangJinLabel</zorder>
  <zorder>faguangLabel</zorder>
  <zorder>guangxianLabel</zorder>
  <zorder>xingxingLabel</zorder>
  <zorder>xunzhangLabel</zorder>
  <zorder>pianmingLabel</zorder>
  <zorder>mingrentangLabel</zorder>
  <zorder>huizhanglishiLabel</zorder>
  <zorder>zhengzaijinxingLabel</zorder>
  <zorder>yuanguangTongLabel</zorder>
  <zorder>yuanguangJinLabel</zorder>
  <zorder>jinkuangLabel</zorder>
  <zorder>yuanguangYinLabel</zorder>
  <zorder>tongkuangLabel</zorder>
  <zorder>jinkuangxiaLabel</zorder>
  <zorder>yinkuangxiaLabel</zorder>
  <zorder>yinkuangLabel</zorder>
  <zorder>tongkuangxiaLabel</zorder>
  <zorder>shuziJin4Label</zorder>
  <zorder>shuziTong2Label</zorder>
  <zorder>shuziYin4Label</zorder>
  <zorder>shuziYin5Label</zorder>
  <zorder>shuziTong6Label</zorder>
  <zorder>shuziJin6Label</zorder>
  <zorder>shuziYin6Label</zorder>
  <zorder>shuziTong3Label</zorder>
  <zorder>shuziTong1Label</zorder>
  <zorder>shuziYin3Label</zorder>
  <zorder>shuziYin7Label</zorder>
  <zorder>shuziJin1Label</zorder>
  <zorder>shuziJin2Label</zorder>
  <zorder>shuziYin2Label</zorder>
  <zorder>shuziTong5Label</zorder>
  <zorder>shuziYin1Label</zorder>
  <zorder>shuziJin7Label</zorder>
  <zorder>shuziTong7Label</zorder>
  <zorder>shuziTong4Label</zorder>
  <zorder>shuziJin5Label</zorder>
  <zorder>shuziJin3Label</zorder>
  <zorder>jiLabel</zorder>
  <zorder>yueLabel</zorder>
  <zorder>zhouLabel</zorder>
  <zorder>qiehuantouxiangJinLabel</zorder>
  <zorder>qiehuantouxiangTongLabel</zorder>
  <zorder>qiehuantouxiangYinLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
